pkg/k8s/wrapper.go

```diff
@@ -260,7 +260,8 @@ func (k *k8sWrapper) CreateCNINode(node *v1.Node, clusterName string) error {
 		},
 		Spec: rcv1alpha1.CNINodeSpec{
 			Tags: map[string]string{
-				fmt.Sprintf(config.VPCCNIClusterNameKey): clusterName,
+				fmt.Sprintf(config.VPCCNIClusterNameKey):      clusterName,
```

Comment:
Did you want to use string formater instead? `fmt.Sprintf("%s: %s", key, value)`?